src/adaptors/ion-protocol/index.js

```diff
@@ -59,7 +60,7 @@ const marketApy = async () => {
       chain: 'ethereum',
       project: 'ion-protocol',
       symbol: market.collateralAssetName + '-' + market.lenderAssetName,
-      tvlUsd: totalSupplyUsd - totalBorrowUsd,
+      tvlUsd: totalSupplyUsd,
```

Comment:
pls revert this line, we use available liq for tvlUsd on our end

Reply:
Hi @slasher125, I modified this line because I noticed a lot of other existing protocols on Defi Llama like Morpho Blue have just their total supply as the tvl. So if possible I would like to keep it this way, if those protocols are not affected by this rule I can revert the change. Let me know your thoughts

Reply:
Just for reference [here](https://github.com/DefiLlama/yield-server/blob/f89fc8dce6703e5b42ff7e76bb2da3350c4b7d86/src/adaptors/morpho-blue/index.js#L424C33-L424C47) is a link to the relevant code in Morpho's adapter, it can also be confirmed by comparing the tvl value on the yield tab on Defi Llama to their market page on their own website.

Reply:
yeah, for morpho its somewhat different given they route to aave if they dont have any liq available.
still, needs to be changed to totalSupplyUsd - totalBorrowUsd. ty

Reply:
I see, I have changed the tvl to total supplied - total borrowed in my recent commit